src/components/post-filter/post-sort.component.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import {
+  SORT_BY_NEWEST,
+  SORT_BY_OLDEST,
+  SORT_BY_MOST_POPULAR,
+  SORT_BY_LEAST_POPULAR,
+} from "database/utils";
+
+const PostSort = ({ handleSort }) => {
+  return (
+    <div className="flex justify-center items-center">
+      <div className="text-grayy p-2"> Sort By</div>
+      <div className="flex justify-between">
+        <form onChange={handleSort}>
```

Comment:
Using an `onChange` on the `<form>` element is how you avoid selecting from DOM. The browser makes the information available to you in the form of an `event` object